src/components/TableHeader/TableHeader.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import "./TableHeader.css";
+
+const TableHeader = props => {
+  const [sortAsc, setSortAsc] = useState(true);
```

Comment:
Ooo look at you fancy boy with your react hooks!

Reply:
Yeahhhh. Right now there is only one variable for asc/desc, but in the future we should have one for each column.